### PR TITLE
Remove Dockerhub image test

### PR DESCRIPTION
**Pull request by lmzuccarelli**
- As we have a quay.io test, this is redundant
- Its causing errors in the CI pipeline due to docker's new rate limiting process

### CODE REVIEW

File: _output/named-golang-versions [hunk 1]
```diff
@@ -0,0 +1,3 @@
+images/Dockerfile.rhel: 1.16
```
Comment (sgreene570) [inline _output/named-golang-versions:1]: Is it necessary to commit the artifacts in `_output`?

Comment (lmzuccarelli) [reply _output/named-golang-versions:1]: I will remove the folder. Thanks for pointing it out

**Comment by sttts** [comment]
This need review from the owning team first.

**Comment by sttts** [comment]
Please find somebody in the test owning team first for lgtm. Maybe @dmage ?

**Pull request merged**